scripts/github_runner/start_runners.sh

```diff
@@ -21,24 +21,78 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-if [ -z $1 ] || [ -z $2 ] || [ -z $3 ]; then
+if [ -z $1 ] || [ -z $2 ] || [ -z $3 ] || [ -z $4 ]; then
     echo "Parameters missing"
-    echo "USAGE: start_runners.sh <num of runners> <Github Access key> <runner label(comma separated)>"
+    echo "USAGE: start_runners.sh <num of runners> <Github URL> <Github Access key> <runner label(comma separated)>"
     exit -1
 fi
 
+# fetch runner token using access token
+ACCESS_TOKEN=$3
+API_VERSION=v3
+API_HEADER="Accept: application/vnd.github.${API_VERSION}+json"
+AUTH_HEADER="Authorization: token ${ACCESS_TOKEN}"
+
+_SHORT_URL=$2
+REPO_NAME="$(echo "${_SHORT_URL}" | grep / | cut -d/ -f4-)"
+_FULL_URL="https://api.github.com/repos/${REPO_NAME}/actions/runners/registration-token"
+
+RUNNER_TOKEN="$(curl -XPOST -fsSL \
+  -H "${AUTH_HEADER}" \
+  -H "${API_HEADER}" \
+  "${_FULL_URL}" \
+| jq -r '.token')"
+
+# install kind from ease-lab/kind
+rm -rf $HOME/kind/
+git clone -b custom_docker_params_for_vHive https://github.com/ease-lab/kind $HOME/kind/
```

Comment:
this should be moved to the setup_host script

Reply:
please also install the kind binary in /usr/local/bin` or so (make sure it's in $PATH)

Reply:
also, please remove the kind folder after installing the binary. Hence, it would make sense to clone the repo to /tmp/ instead of the home folder

Reply:
done

Reply:
kind moved to /tmp/ and binary moved to /usr/local/bin/